README.md

```diff
@@ -57,6 +57,7 @@ used within the yaml property:
   - `commentPrefix` - a string that defines prefix for context comment for
     translators. Only comments that start with the provided string will
     be extracted and added to ResultSet, all other are ignored.
+- `checkTranslatability`: by default the plugin checks if lines in yaml file are translatable, to turn that check off `checkTranslatability` setting has to set to false.         
```

Comment:
This description feels like translatability is a property of a yaml value that can (should?) be set somehow (as if one had to include some tag in a comment like `::l10n::`). How about something like this:
> by default, the plugin tries to guess which values in the file are strings for translation and filter out others; setting `checkTranslatablility: false` will disable this behaviour and all values from the file will be extracted.

Reply:
Right, thanks!